src/services/auth/oauth2/types.ts

```diff
@@ -0,0 +1,70 @@
+import {
+  AffectedRecords,
+  OptionsBase,
+  OptionsWithRql,
+  PagedResult,
+} from '../../types';
+
+export interface AuthOauth2Service {
+  /**
+   * Create an OAuth2 authorization
+   *
+   * Permission | Scope | Effect
+   * - | - | -
+   * none | | Everyone can use this endpoint
+   * @see https://swagger.extrahorizon.com/swagger-ui/?url=https://swagger.extrahorizon.com/auth-service/2.0.4-dev/openapi.yaml#/OAuth2/post_oauth2_authorizations
```

Comment:
I think all of these links are outdated. 🤔

Reply:
Yes, first wanted to do the restructuring. I will tackle the inline docs including links next